src/types/das-types.ts

```diff
@@ -0,0 +1,251 @@
+// Types for DAS
+
+import {
+  AssetSortBy,
+  AssetSortDirection,
+  DASInterface,
+  DASScope,
+  DasContext,
+  OwnershipModel,
+  RoyaltyModel,
+  UseMethods,
+} from "./enums";
+
+export type DASOption<T> = T | null;
```

Comment:
Isn't it technically undefined? So instead of using `foo: DASOption<string>` you can just say `foo?: string`.